README.md

```diff
@@ -158,6 +183,20 @@ make rpm-packages #build rpm packages
 
 To build `.rpm` packages with tezos binaries.
 
+### Publish packages on Fedora Copr
+
+In order to publish packages on Copr you will have to build source packages,
+you can do this by running the following command:
+```
+nix-build -A rpm-source-packages -o rpm-source-packages \
+--arg timestamp "$(date +\"%Y%m%d%H%M\")"
+# Copy files from /nix/store
+mkdir -p source-packages
+cp rpm-source-packages/* source-packages
+```
+After this `source-packages` directory will contain `.src.rpm` packages which
+can be uploaded to Copr repository.
```

Comment:
Should it be done from web UI? Is there CLI for it?

It would be good to have a script that one can run whenever they make a new release (from a pre-prelease) and it would upload everything to ppa/copr.

Reply:
I think that when I was researching this stuff, I haven't found any API for Copr, but I'll take a look at it once again

Reply:
> It would be good to have a script that one can run whenever they make a new release (from a pre-prelease) and it would upload everything to ppa/copr.

The problem is that PPA requires some ubuntu specific binaries (`debsign` and `dput`) which are not presented in other Linux distros and nixpkgs :(

I'm afraid, that if there is some API for Copr, it can be Fedora specific as well

Reply:
Right, there is https://docs.pagure.org/copr.copr/src_rpm_upload.html

Reply:
Hm, okay.